docs/guides/testing-functions.md

```diff
@@ -53,8 +53,8 @@ If `FooComponent` uses the `foo` function as a callback:
         <!-- Field with a callback we want to test -->
         <field id="myField" type="string" onChange="foo" />
 
-        <!-- Field with a callback we want to test -->
-        <field id="sideEffectField" type="string">
+        <!-- Field that gets modified as a side-effect of the `foo` function -->
```

Comment:
This fixes a different copy-lasagna error from #85... turns out when i write documentation i need to be a little more careful 😅